### DISCUSSION TITLE
OKD 4.11 | Bootstrap issues | failed at release-image-pivot.service

### DISCUSSION THREAD

**Discussion by vjaincatalogic** [okd-project/okd#1467] [General]
**Description**

I see following error while checking the release-image-pivot.service logs

[core@bootstrap ~]$ journalctl -b -f -u release-image-pivot.service
Jan 16 10:46:40 bootstrap.vjain-okd.vjain.lan podman[2016]: 2023-01-16 10:46:40.161355132 +0000 UTC m=+0.232406258 container remove 3491dd6fec3593a6168181dc741554429ad0ba84bf39c07d8bc0914e3514b94b (image=quay.io/openshift/okd@sha256:924958ed5ba93ea9d72e4b5a319d0d4df220edbb891816c106215cbf318a0a40, name=zen_chaplygin, io.openshift.release=4.11.0-0.okd-2022-12-02-145640, io.openshift.release.base-image-digest=sha256:6584aa581e369eceb39188a14669b78040e5408e544cd499385b2cbee6278e00)
Jan 16 10:47:03 bootstrap.vjain-okd.vjain.lan bootstrap-pivot.sh[2053]: Error: writing blob: storing blob to file "/var/tmp/storage1480818424/1": happened during read: unexpected EOF
Jan 16 10:47:03 bootstrap.vjain-okd.vjain.lan bootstrap-pivot.sh[1882]: Pull failed. Retrying quay.io/openshift/okd-content@sha256:fa1104b5fd668474d8787ddde8de9d9a1cad86ff3d6b478d988e32b61e43415a...
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan bootstrap-pivot.sh[2976]: dc6adb5f9c3536c24de8f424c82d04c0c74db871c60ae58de536f3e42969e1d9
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan podman[2976]: 2023-01-16 10:47:05.041916684 +0000 UTC m=+1.795650271 image pull  quay.io/openshift/okd-content@sha256:fa1104b5fd668474d8787ddde8de9d9a1cad86ff3d6b478d988e32b61e43415a
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan bootstrap-pivot.sh[1882]: attempt to record the end of a stage without starting one
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan systemd[1]: release-image-pivot.service: Main process exited, code=exited, status=1/FAILURE
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan systemd[1]: release-image-pivot.service: Failed with result 'exit-code'.
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan systemd[1]: Failed to start release-image-pivot.service - Pivot bootstrap to the OpenShift Release Image.
Jan 16 10:48:01 bootstrap.vjain-okd.vjain.lan systemd[1]: release-image-pivot.service: Consumed 1min 29.036s CPU time.

**Version**
OKD-version: 4.11.0-0.okd-2022-12-02-145640
UPI Method

**How reproducible**
1. Followed the required steps to produce ignition file for bootstrap node from the following document:
https://docs.okd.io/4.11/installing/installing_vsphere/installing-vsphere.html#machine-vsphere-machines_installing-vsphere

2. Created a VM with all the prerequisites and started it.

**Log bundle**
[log-bundle-20230117124519.tar.gz](https://github.com/okd-project/okd/files/10432346/log-bundle-20230117124519.tar.gz)

**Comment by vrutkovs** [comment]
>Jan 16 10:47:03 bootstrap.vjain-okd.vjain.lan bootstrap-pivot.sh[2053]: Error: writing blob: storing blob to file "/var/tmp/storage1480818424/1": happened during read: unexpected EOF

Is this reproducible or a one-off storage issue?

**Comment by vjaincatalogic** [comment]
Okie, Let me check by creating new cluster from scratch.

**Reply by osirisstar-hub** [reply]
any update on this issue.  we tried and got the same thing.
